source/AMDSCentralServer.cpp

```diff
@@ -133,3 +108,36 @@ void AMDSCentralServer::onHundredMillisecondTimerUpdate(){
 
 	amptek1BufferGroup_->append(oneSpectralDataHolder);
 }
+
+void AMDSCentralServer::initializeBufferGroup(quint64 maxCountSize)
+{
+	QList<AMDSAxisInfo> mcpBufferGroupAxes;
+	mcpBufferGroupAxes << AMDSAxisInfo("X", 1024, "X Axis", "pixel");
+	mcpBufferGroupAxes << AMDSAxisInfo("Y", 512, "Y Axis", "pixel");
+	AMDSBufferGroupInfo mcpBufferGroupInfo("AFakeMCP", "Fake MCP Image", "Counts", mcpBufferGroupAxes);
+	AMDSBufferGroup *mcpBufferGroup = new AMDSBufferGroup(mcpBufferGroupInfo, maxCountSize);
+	AMDSThreadedBufferGroup *mcpThreadedBufferGroup = new AMDSThreadedBufferGroup(mcpBufferGroup);
+	bufferGroups_.insert(mcpThreadedBufferGroup->bufferGroupInfo().name(), mcpThreadedBufferGroup);
+
+	QList<AMDSAxisInfo> amptek1BufferGroupAxes;
+	amptek1BufferGroupAxes << AMDSAxisInfo("Energy", 1024, "Energy Axis", "eV");
+	AMDSBufferGroupInfo amptek1BufferGroupInfo("Amptek1", "Amptek 1", "Counts", amptek1BufferGroupAxes);
+	amptek1BufferGroup_ = new AMDSBufferGroup(amptek1BufferGroupInfo, maxCountSize);
+	AMDSThreadedBufferGroup *amptek1ThreadedBufferGroup = new AMDSThreadedBufferGroup(amptek1BufferGroup_);
+	bufferGroups_.insert(amptek1ThreadedBufferGroup->bufferGroupInfo().name(), amptek1ThreadedBufferGroup);
+
+	AMDSBufferGroupInfo energyBufferGroupInfo("Energy", "SGM Beamline Energy", "eV");
+	energyBufferGroup_ = new AMDSBufferGroup(energyBufferGroupInfo, maxCountSize);
+	AMDSThreadedBufferGroup *energyThreadedBufferGroup = new AMDSThreadedBufferGroup(energyBufferGroup_);
+	bufferGroups_.insert(energyThreadedBufferGroup->bufferGroupInfo().name(), energyThreadedBufferGroup);
+}
+
+void AMDSCentralServer::startTimer()
+{
+	AMDSErrorMon::information(this, 0, "Starting the timer to update data buffer ...");
```

Comment:
If this is going to stay, please make a normal error code rather than leaving it at 0.